### ISSUE TITLE
Refactor build-yocto-genivi to implement a GoCD Agent

### ISSUE THREAD

**Issue by gmacario** [gmacario/easy-build#198]
Contributions by [GoCD](http://www.go.cd/) gurus welcome...

See related https://at.projects.genivi.org/jira/browse/TOOL-21

**Comment by gunnarx** [comment]
A useful idea - just make sure you check here:  
Looks like it's done already, or can easy_build improve upon it?  
https://github.com/gocd/gocd-docker

**Comment by gmacario** [comment]
Opened a RFC Pull Request at https://github.com/gmacario/easy-build/pull/203

**Comment by gmacario** [comment]
Merged PR #203 

Will close this issue as soon as someone can test it against go.genivi.org

**Comment by gmacario** [comment]
Successfully tested go-agent inside Docker container

```
$ mkdir -p ~/test-goagent && cd ~/test-goagent
$ docker run -ti \
  --hostname=test-genivigo-agent1 \
  --user=go \
  --workdir=/var/lib/go-agent \
  --volume $PWD:/home/build \
  gmacario/build-yocto-genivi
```

Logged as go@container

```
$ vi /etc/default/go-agent
$ service go-agent start
```

CC @gunnarx
